gateway/handler.go

```diff
@@ -252,6 +261,21 @@ func newHandler(c Config, api API) *handler {
 			"gw_raw_block_get_duration_seconds",
 			"The time to GET an entire raw Block from the gateway.",
 		),
+		// TAR: time it takes to return requested TAR stream
+		tarStreamGetMetric: newHistogramMetric(
+			"gw_tar_stream_get_duration_seconds",
+			"The time to GET an entire TAR stream from the gateway.",
+		),
+		// Codec: time it takes to return requested DAG-JSON/-CBOR document
+		codecDocumentGetMetric: newHistogramMetric(
+			"gw_codec_get_duration_seconds",
+			"The time to GET an entire DAG-JSON/CBOR file from the gateway.",
+		),
```

Comment:
Better to name these metrics very clearly, and limit this one to json/cbor.

I don't think we will see new codecs any time soon, and even when they are added, and these will be handled by different response types (and probably something-something WASM) :)

```suggestion
		// JSON/CBOR: time it takes to return requested DAG-JSON/-CBOR document
		jsoncborDocumentGetMetric: newHistogramMetric(
			"gw_jsoncbor_get_duration_seconds",
			"The time to GET an entire DAG-JSON/CBOR block from the gateway.",
		),
```